src/deploy/jinja2_templates/dell-environment-edge.j2.yaml

```diff
@@ -11,11 +11,22 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 resource_registry:
-  OS::TripleO::NodeUserData: ./first-boot.yaml
+  OS::TripleO::NodeUserData: {{node_user_data_path}}
 
 parameter_defaults:
+  NeutronPublicInterface: bond1
+  NeutronNetworkType: vlan
+  NeutronNetworkVLANRanges: physint:201:250,physext
```

Comment:
Should that range be hardcoded here ? Should probably be set from ini's tenant_vlan_range

Reply:
The following are hard-coded in pilot/templates/dell-environment.yaml and never modified by automation
  NeutronPublicInterface: bond1
  NeutronNetworkType: vlan
  NeutronBridgeMappings: physint:br-tenant,physext:br-ex

The attribute below is updated before the overcloud deployment in pilot/dell_nfv.py,  I missed that we use that module for all deployments, not just NFV type deployments.
NeutronNetworkVLANRanges: physint:201:250,physext

I'll populate those variables with Jinja2, we may want to retrofit dell_nfv.py to also set NeutronNetworkType and NeutronPublicInterface.

Reply:
+1, will do

Reply:
Updated to get attribute from tenant_vlan_range, will be in next patch set.